internal/pkg/deploy/cloudformation/app.go

```diff
@@ -477,6 +477,7 @@ func (cf CloudFormation) getLastDeployedAppConfig(appConfig *stack.AppStackConfi
 	if err != nil {
 		return nil, fmt.Errorf("parse previous deployed stackset %w", err)
 	}
+	previouslyDeployedConfig.App = appConfig.Name
```

Comment:
Why did we make this change? How come it wasn't an issue before

Reply:
because otherwise [`.App`](https://github.com/aws/copilot-cli/blob/mainline/templates/app/versions/v1.0.1/cf.yml#L3) will be an empty string. This wasn't an issue because we take different routes like this https://github.com/aws/copilot-cli/blob/b538de9a5877dc7714124c98c24e6da942432ebb/internal/pkg/deploy/cloudformation/app.go#L264-L269. but the function itself based on its name i think should have the correct app name.